packages/cli/src/oclif/commands/register.js

```diff
@@ -28,6 +33,12 @@ class RegisterCommand extends ZapierBaseCommand {
       );
     }
 
+    if ('title' in this.args && this.args.title.length < MIN_TITLE_LENGTH) {
```

Comment:
@pragmatic-zac When running `zapier register` (without an argument, thus triggering this flow), I am getting: `Error: Cannot read properties of undefined (reading 'length')`. Using the `--debug` flag shows that ` zapier:register args are { title: undefined }` so it seems that checking for just `'title' in this.args` is not enough 😢

Reply:
That's a really helpful shell alias, thank you!

Made a slight adjustment and this is fixed in my manual tests now. Ready for another look whenever you're able 😄